phone_verify/serializers.py

```diff
@@ -15,35 +15,35 @@
 
 
 class PhoneSerializer(serializers.Serializer):
-    phone_number = PhoneNumberField()
+	phone_number = PhoneNumberField()
 
 
 class SMSVerificationSerializer(serializers.Serializer):
-    phone_number = PhoneNumberField(required=True)
```

Comment:
These changes are irrelevant to the current changes. Can you please remove them?

Reply:
Do you mean changing indents from space to tab?

Reply:
Yes. PEP8 recommends using 4 spaces of indentation at each level.